frontend/src/store/reducers/index.js

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable prettier/prettier */
+import {
+	REGISTER_USER,
+	LOGIN_USER,
+	LOGOUT_USER,
+	ADD_ARTICLE,
+	DELETE_ARTICLE,
+	FETCH_DATA,
+	DATA_FETCHED,
+	ERROR
+} from '../actions';
+
+const initialState = {
+	users: [],
+	fetching: false,
+	success: false,
+	error: null
+};
+
+export const userReducers = (state = initialState, action) => {
+	switch (action.type) {
+		case FETCH_DATA:
+			return { ...state, fetching: true };
+		case ERROR:
+			return { ...state, error: 'Error' + action.err };
+		case DATA_FETCHED:
+			return { ...state, notes: action.payload, fetching: false };
+		case REGISTER_USER:
```

Comment:
If an error happens when will it ever reset to null? Might want to reset fetching in `ERROR`. Another idea for naming cases following this kind of structure:
`FETCHING_DATA` `FETCHED_DATA` `FETCH_ERROR` `ADDING_DATA` `ADDED_DATA` `ADD_ERROR` etc
This way there's consistency in naming?